hivemq-edge/src/frontend/src/components/rjsf/Templates/CompactArrayFieldTemplate.tsx

```diff
@@ -0,0 +1,76 @@
+import { FC } from 'react'
+import { JSONSchema7 } from 'json-schema'
+import { RJSFSchema, ArrayFieldTemplateProps, getTemplate, getUiOptions, ArrayFieldTemplateItemType } from '@rjsf/utils'
+import { Box, HStack, Table, Tbody, Th, Thead, Tr } from '@chakra-ui/react'
+
+import AddButton from '@/components/rjsf/__internals/AddButton.tsx'
+import { AdapterContext } from '@/modules/ProtocolAdapters/types.ts'
+
+export const CompactArrayFieldTemplate: FC<ArrayFieldTemplateProps<unknown, RJSFSchema, AdapterContext>> = (props) => {
+  const { canAdd, disabled, idSchema, uiSchema, items, onAddClick, readonly, registry, required, schema, title } = props
+  const uiOptions = getUiOptions(uiSchema)
+  const ArrayFieldDescriptionTemplate = getTemplate<'ArrayFieldDescriptionTemplate'>(
+    'ArrayFieldDescriptionTemplate',
+    registry,
+    uiOptions
+  )
+  const ArrayFieldItemTemplate = getTemplate<'ArrayFieldItemTemplate'>('ArrayFieldItemTemplate', registry, uiOptions)
+  const ArrayFieldTitleTemplate = getTemplate<'ArrayFieldTitleTemplate'>('ArrayFieldTitleTemplate', registry, uiOptions)
+
+  const { items: SchemaItems } = schema as JSONSchema7
+  const { properties } = SchemaItems as JSONSchema7
+
+  // Better approach to unidentified headers?
+  if (!properties) return null
+
+  return (
+    <Box>
+      <ArrayFieldTitleTemplate
+        idSchema={idSchema}
+        title={uiOptions.title || title}
```

Comment:
should we document somewhere that `uiOptions.title` takes precedence over `title`?

Reply:
It's the expected behaviour of the RJSF library: the `(JSON)schema` provides the bulk of the information (e.g. title, description, validation rules, ...) unless they are overridden by the optional `uiSchema`